src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Startup.cs

```diff
@@ -1,75 +1,79 @@
 ﻿using AutoMapper;
 using DevAdventCalendarCompetition.Extensions;
 using DevAdventCalendarCompetition.Repository.Context;
+using DevAdventCalendarCompetition.Repository.Models;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Localization;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Swashbuckle.AspNetCore.Swagger;
+using System.Globalization;
 
 namespace DevAdventCalendarCompetition
 {
-    public class Startup
-    {
-        private const string DockerEnvName = "Docker";
-        public Startup(IHostingEnvironment env)
-        {
-            var configurationBuilder = new ConfigurationBuilder()
-                .AddJsonFile($"appsettings.{env.EnvironmentName}.json")
-                .AddEnvironmentVariables();
+	public class Startup
```

Comment:
why there are so many empty changes? do you have CodeMaid with default settings?